src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -130,9 +133,12 @@ public partial struct JsonDocumentOptions
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
+        public bool IsImmutable { get { throw null; } }
```

Comment:
I assume here all we are going to do is to return a `_dummy is JsonDocument`? We don't want to actually save a backing field as that would increase the size of the struct